campaign-launcher/interfaceV2/src/components/Campaigns/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC, useEffect, useState } from 'react';
+
+import { Box } from '@mui/material';
+import { useSearchParams } from 'react-router-dom';
+
+import { CampaignsView } from '../../types';
+import ActiveCampaignsFilter from '../ActiveCampaignsFilter';
+import AllCampaigns from '../AllCampaigns';
+import CampaignsViewDropdown from '../CampaignsViewDropdown';
+import JoinedCampaigns from '../JoinedCampaigns';
+import MyCampaigns from '../MyCampaigns';
+
+const Campaigns: FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [campaignsView, setCampaignsView] = useState(() => {
+    if (searchParams.size === 0) {
+      return CampaignsView.ALL;
+    }
```

Comment:
Do we really need this if `ALL` is a fallback anyways?

Reply:
@dnechay early exit